zcash_client_backend/src/data_api.rs

```diff
@@ -1323,6 +1346,14 @@ pub mod testing {
             Ok(None)
         }
 
+        fn insert_address_with_diversifier_index(
+            &mut self,
+            _account: AccountId,
+            _diversifier_index: DiversifierIndex,
+        ) -> Result<UnifiedAddress, Self::Error> {
+            todo!()
```

Comment:
Please either implement this, or add an issue to track the TODO

Reply:
Would returning `Err(())` count? It's difficult to imagine a better implementation given this mock doesn't store anything at all.

Reply:
I was thinking that the mock could store a seed, but just adding an issue for this is fine.